config/production.json

```diff
@@ -133,6 +133,7 @@
 		"ssr/log-prefetch-errors": true,
 		"ssr/prefetch-timebox": true,
 		"ssr/sample-log-cache-misses": true,
+		"stats/fixed-nav-headers": false,
```

Comment:
You probably don't need to specify this here (nor horizon, stage, test, or wpcalypso) since the flag will default to false when undefined 🙂

Reply:
I was working based [on this line](https://github.com/Automattic/wp-calypso/blob/8d0daadfe366c8c31905eb65ca7c2f2aa74424d9/config/README.md#feature-flags) from the docs:

> Please make sure to add new feature flags alphabetically so they are easy to find and add any new feature flags in all config files (client.json, development.json, production.json, horizon.json, stage.json, test.json, wpcalypso.json).

I'm happy to remove it though. Will get that change in today.

Reply:
Ah shoot, I'm in the wrong here, then. Thanks for letting me know — and feel free to revert the suggested change if you'd prefer.

Reply:
No worries! I'll get those changes back in there.